blueprints/third-party-solutions/phpipam/glb.tf

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  glb_create   = var.phpipam_exposure == "EXTERNAL"
+  iap_sa_email = try(module.project.service_accounts.robots["iap"].email, "")
+}
+
+# Reserved static IP for the Load Balancer
+module "addresses" {
+  source           = "../../../modules/net-address"
+  project_id       = var.project_id
```

Comment:
I think you also want `count = local.glb_create ? 1 : 0` here, otherwise you reserve the address but never use it

Reply:
Done!